spec/JSON-RPC.md

```diff
@@ -1727,6 +1727,29 @@ Errors: `Invalid Params`
 
 [Back to **List of methods**](#list-of-methods)
 
+## mempool_deleteAllPendingTransactions
+Delete all pending transactions in both current and future queue.
```

Comment:
We are using a third-person singular verb in the description. We are assuming that the method name is the subject.

Reply:
Ok